src/Data/Nat/Properties.agda

```diff
@@ -1617,6 +1621,26 @@ module ≤-Reasoning where
     public
     hiding (_≈⟨_⟩_)
 
+
+open ≤-Reasoning using () renaming (begin_ to begin≤_; _∎ to _end≤;
+                                    _≡⟨_⟩_ to _≡≤[_]_; _≤⟨_⟩_ to _≤[_]_)
+
+m<m*n :  ∀ {m n} → 0 < m → 1 < n → m < m * n
+m<m*n {m} {n} 0<m 1<n =
```

Comment:
This proof will get much simpler if you pattern match on `m` on the first line as follows:
```
m<m*n :  ∀ {m n} → 0 < m → 1 < n → m < m * n
m<m*n {suc m} {n} (s≤s 0≤m) 1<n =
```

Reply:
I shall look into this.